src/vue/modules/pre-issuance-form/wrappers/issuance.js

```diff
@@ -0,0 +1,8 @@
+export class Issuance {
```

Comment:
useless record, instead better deifne the field withing the `data` property, like: 
```js
data: _ => ({
  issuance: {
    amount: 0,
    asset: '',
    xdr: '',
    isUsed: false,
  }
})
```

Reply:
This record is used as a prop type by `issuance-viewer` component. Should we just pass `issuanceAmount` and `issuanceAsset` to it in that case?